src/main/java/polyglot/CommandLineArgs.java

```diff
@@ -47,6 +47,9 @@
   @Option(name = "--oauth2_token_endpoint")
   private String oauth2TokenEndpoint;
 
+  @Option(name = "--oauth2_access_token")
```

Comment:
In "initialize", could you please validate that we are not passing an access token *and* a refresh token flag